modules/contact/doc/content/modules/contact/index.md

```diff
@@ -117,7 +118,10 @@ residuals. The third column denotes the type of non-linear complimentarity
 problem (NCP) function used to ensure that the contact constraints are
 satisfied. Min indicates the canonical min function (see
 [std::min](https://en.cppreference.com/w/cpp/algorithm/min)); FB represents the
-Fischer-Burmeister function. The fourth column is the simulation time in
+Fischer-Burmeister function. `RANFS` denotes the Reduced Active Nonlinear
```

Comment:
`RANFS denotes the [Reduced Active Nonlinear Function Set](RANFSNormalMechanicalContact.md) scheme` ?